packages/cozy-harvest-lib/src/services/budget-insight.spec.js

```diff
@@ -368,7 +368,7 @@ describe('handleOAuthAccount', () => {
       t,
       account: {
         ...account,
-        ...{ auth: { bankId: TEST_BANK_BI_ID } },
+        ...{ auth: { bankId: TEST_BANK_COZY_ID } },
```

Comment:
TEST_BANK_LINXO_ID ?

Reply:
I actually prefer to replace all references to Linxo by Cozy